core/src/main/kotlin/apiDocs/RepositoriesDocs.kt

```diff
@@ -745,3 +747,37 @@ val deleteUserFromRepositoryGroup: OpenApiRoute.() -> Unit = {
         }
     }
 }
+
+val getUsersForRepository: OpenApiRoute.() -> Unit = {
+    operationId = "GetUsersForRepository"
+    summary = "Get all users that have rights for a repository, grouped by user privilege."
+
+    request {
+        pathParameter<Long>("repositoryId") {
+            description = "The repository's ID."
+        }
+    }
+
+    response {
+        HttpStatusCode.OK to {
+            description = "Success"
+            jsonBody<Map<UserGroup, Set<User>>> {
+                example("Get users for repository") {
+                    value = mapOf(
+                        Pair(
+                            UserGroup.READERS,
+                            setOf(
+                                User(
+                                    username = "jdoe", firstName = "John", lastName = "Doe", email = "johndoe@mail.ee"
```

Comment:
> johndoe@mail.ee

Please don't use a mail address that possibly exists.
Please use an `example.com` domain.